components/src/core/drawer/drawer-nav-item.tsx

```diff
@@ -118,10 +118,14 @@ const makeStyles = (
     flipIcon: ViewStyle;
 }> => {
     // Primary color manipulation
-    const fivePercentOpacityPrimary = color(theme.colors.primaryBase || theme.colors.primary)
+    const fivePercentOpacityPrimary = color(
+        (theme.dark ? theme.colors.primaryPalette?.dark : theme.colors.primaryPalette?.main) || theme.colors.primary
+    )
         .fade(0.95)
```

Comment:
yo we should replace these 'fade's with 'alpha's. 
fade(0.95) -> alpha(0.5) 
fode(0.8) -> alpha(0.2)
etc.

(easier to read that way)
We originally used fade because we didn't know the alpha function existed.